index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/8b2919a184.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!-- ------------------The Nav Bar-------------------->
+    <header class="parent-container header">
+      <a href="#" class="flex-item-1">My Logo</a>
+      <nav>
+        <a href="#"><i class="fa-solid fa-bars flex-item-2"></i></a>
+      </nav>
+    </header>
+    <!-- -------------------------------The Home Page---------------------------------- -->
+
+    <section class="main-page">
+      <div class="home-page-details">
+        <h1>I'm Amen<br />Glad to see you!</h1>
```

Comment:
- [ ] [OPTIONAL ] This design is to be fixed, please consider aligning the text here as shown on the left.